.github/workflows/minor_version_release.yml

```diff
@@ -0,0 +1,41 @@
+name: Minor Version Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release (in the format of v1.10.x) '
+        required: true
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.8.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8.10
```

Comment:
I saw that we only specify version 3.8 here:https://github.com/deepset-ai/haystack/blob/b84a6b17165dbf10665794b5decdefa113300749/.github/workflows/tests.yml#L42 but 3.8.10 here: https://github.com/deepset-ai/haystack/blob/2b803a265b6e66a46ec7e6688a2fd2b7b3114d77/.github/workflows/readme_integration.yml#L22
Let's use one version consistently if there is no specific reason for different versions.

Reply:
Yes good point - I have made that change now. The sed command was tested in another branch.